Source/Core/BoogiePL.atg

```diff
@@ -27,11 +27,22 @@ readonly List<Expr>/*!*/ dummyExprSeq;
 readonly TransferCmd/*!*/ dummyTransferCmd;
 readonly StructuredCmd/*!*/ dummyStructuredCmd;
 
+public static int ParseLibraryDefinitions(out Program program)
+{
+	string libraryDefinitionsFileName = "LibraryDefinitions.bpl";
+	Assembly asm = Assembly.GetExecutingAssembly();
+	var resourceName = asm.GetManifestResourceNames().First(s => s.EndsWith(libraryDefinitionsFileName, StringComparison.CurrentCultureIgnoreCase));
```

Comment:
I would remove this search and just directly use the resource name in `GetManifestResourceStream` below. On my machine the name is `Core-NetCore.LibraryDefinitions.bpl`. Is it the same for you? I don't quite get why it is not `Boogie.Core.LibraryDefinitions.bpl`, because I thought the prefix is the assembly name. But somehow the prefix is taken from the name of the `csproj` file.

If you do this change, you can also drop `using System.Linq;` from `Parser.frame`.

And the whole method `ParseLibraryDefinitions` uses tabs instead of spaces for indentation.